Source/Scene/Primitive.js

```diff
@@ -1119,57 +1121,79 @@ define([
                 });
             }
 
+            var transferTerrainHeightsPromise;
             if (!defined(createGeometryTaskProcessors)) {
                 createGeometryTaskProcessors = new Array(numberOfCreationWorkers);
                 for (i = 0; i < numberOfCreationWorkers; i++) {
                     createGeometryTaskProcessors[i] = new TaskProcessor('createGeometry', Number.POSITIVE_INFINITY);
                 }
+                // Push tasks transferring the JSON to the workers.
+                transferTerrainHeightsPromise = ApproximateTerrainHeights.initialize()
+                    .then(function() {
+                        var terrainHeightsString = JSON.stringify(ApproximateTerrainHeights._terrainHeights);
```

Comment:
Are we sure this is a good idea, did we time this? I’m not sure why we didn’t just make everything work on the worker

Reply:
It's not terrible, even on my chromebook-grade machine this is typically under 50 ms according to simple `console.time` wrapping. I think it's swallowed by the rest of worker spinup, if that's anywhere near as slow as it is in Node. Keep in mind that the JSON parse on the worker side is something that has to happen even if the worker's requesting the terrain heights instead.

We could also hold the string on the main thread alongside the parsed version, then release it after workers are spun up... I'll go ahead and add that right after  I get CI passing on Travis again.

Reply:
If you only timed the Stringify, then you are missing the part where it gets sent over to the worker.  For a string, that's probably not a lot of overhead though.  However, why wouldn't just making getModuleUrl work in workers be a better solution?  I feel like this another bug like this will pop up again in the future the next time we want to do something with a url on a worker.

Reply:
> However, why wouldn't just making getModuleUrl work in workers be a better solution? I feel like this another bug like this will pop up again in the future the next time we want to do something with a url on a worker.

I guess this would also get us around the test failure that I keep running into on Travis in this PR...